### PR TITLE
Update kubernetes_e2e.py to fix typo

### PR DESCRIPTION
**Pull request by jianliao82**
fix typo GETing -> Getting

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -549,7 +549,7 @@ def set_up_aws(workspace, args, mode, cluster, runner_args):
     mode.add_aws_runner()
 
 def read_gcs_path(gcs_path):
-    """reads a gcs path (gs://...) by GETing storage.googleapis.com"""
```
Comment (BenTheElder) [inline scenarios/kubernetes_e2e.py]: this is intentionally because it does an HTTP `GET` request

Comment (BenTheElder) [reply scenarios/kubernetes_e2e.py]: It probably should say `GET-ing` or something slightly more verbose like `by HTTP GET to storage.googleapis.com`

Comment (jianliao82) [reply scenarios/kubernetes_e2e.py]: thanks a lot! Already updated it.

**Comment by jianliao82** [comment]
@shyamjvs  needs-ok-to-test  ?

**Pull request merged**